docs/docs/reference/tasty-reflect.md

```diff
@@ -38,7 +38,7 @@ It will also import all extractors and methods on TASTy Reflect trees. For examp
 ```scala
 def natConstImpl(x: Expr[Int])(implicit reflection: Reflection): Expr[Int] = {
   import reflection._
-  val xTree: Term = x.reflect
+  val xTree: Term = x.unseal
```

Comment:
This is not the only place in the documentation that need to be update (line 35 in this file for example). I would search globally for reflect/reify

Reply:
Done